src/blocks/media-block/image-block/CleanImageBlock.tsx

```
@@ -0,0 +1,621 @@
+    })
+    .filter(Boolean)
+    .join(', ');
+};
+
+// Generate sizes attribute
+const generateSizes = (sources: ImageSource[]): string => {
+  if (!sources || !Array.isArray(sources)) return '';
+  
+  return sources
+    .map(src => {
+      if (src && src.media && isNonEmptyString(src.media)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)